cbdcconfig/defaults.go

```diff
@@ -6,4 +6,5 @@ const (
 	DEFAULT_GCP_REGION       = "us-west1"
 	DEFAULT_CAPELLA_ENDPOINT = "https://api.cloud.couchbase.com"
 	DEFAULT_CAPELLA_PROVIDER = "aws"
+	DEFAULT_HOST_NAME        = "https://cb-engineering.s3.amazonaws.com/"
```

Comment:
Might be worth adding some detail to this hostname. What is it used for specifically or the hostname of etc

Reply:
Additionally, this "DEFAULT_HOST_NAME" should be used to initialize some configurable aspect of the Capella config, and since this project is public, the default should probably be a blank string, but it would be overrideable for our internal environments.